tests/unit/dojson/test_dojson_institutions.py

```diff
@@ -30,6 +30,41 @@
 from inspirehep.dojson.utils import clean_record
 
 
+def test_superseded_institutions():
```

Comment:
This test should be called `test_name_from_110__a_t_u_double_x_double_z`. Also it should be closer to the other tests that talk about the `name` rule, a bit lower in the file.